### DISCUSSION TITLE
RocketMQ 5.0版本，DLedger Controler模式是否支持多主

### DISCUSSION THREAD

**Discussion by onejimmyboy** [apache/rocketmq#7071] [General]
通过官网和[5.1.0]关于Dledger和controller疑问 #6172这个discussion可以看出来，controller是支持1主1从，1主多从
那是否支持多主多从呐？
2组broker集群，是否可以同时使用一组controller呐？

**Comment by RongtongJin** [comment]
@onejimmyboy 支持。多组broker都可以是哟哦那个同一组controller。
